### PR TITLE
Fix document structure to clean the directory

### PR DESCRIPTION
**Pull request by kchygoe**
## Motivation and Context
I want to clean the top directory by move the requirements.txt to docs dir to simplify the repo. It is easier to understand the structure.

## Description

make docs directory more independent. 

## How has this been tested?
- locally `make html`
- (tbd) netlify

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,12 +7,14 @@ jobs:
     steps:
       - checkout
       # Build documents
-      - run: sudo pip install -r doc.requirements.txt && cd docs && make html
+      - run:
+          command: |
+            sudo pip install -r docs/requirements.txt
```
Comment (ruimashita) [inline .circleci/config.yml]: Why does the command `sudo pip install -r docs/requirements.txt` need ?
`make html` include pip install, I think.

Comment (kchygoe) [reply .circleci/config.yml]: I thought too, but circleci/python requires `sudo` to install pip packages in /usr/local

Comment (ruimashita) [reply .circleci/config.yml]: I see 😄 Please write the comment something like `#circleci/python requires sudo to install pip packages`.

Comment (kchygoe) [reply .circleci/config.yml]: I added comments

**Comment by ruimashita** [comment]
😭 fail DLK test ... something wrong.

**Comment by ruimashita** [comment]
run dlk test

**Comment by kchygoe** [comment]
dlk jenkins test

**Comment by kchygoe** [comment]
run dlk test

**Pull request merged**